Libs/DICOM/Widgets/ctkDICOMImage.cpp

```diff
@@ -119,7 +119,7 @@ QImage ctkDICOMImage::frame(int frame) const
     const unsigned long length = width * height + offset;
     /* create output buffer for DicomImage class */
     QByteArray buffer;
-    buffer.append(header);
+    buffer.append(header.toLatin1());
```

Comment:
@lassoan Should this be the following per some of the changes you made in https://github.com/commontk/CTK/commit/daa513fdc86347c7ec7712974991db0fd8343a32?
```suggestion
    buffer.append(header.toUtf8());
```

Reply:
Thanks for reviewing.

I will update accordingly